core/server/src/main/webapp/browse-body.jsp

```diff
@@ -38,6 +65,10 @@
               <% } %>
             </ul>
```

Comment:
Thank your for your modify suggestion. It is look good better than mine. I put it inside the `<div class="navbar-inner">` soon.